src/kiwixapp.h

```diff
@@ -63,6 +64,7 @@ class KiwixApp : public QApplication
     KiwixApp(int& argc, char *argv[]);
     virtual ~KiwixApp();
     static KiwixApp* instance();
+    static QString gt(const QString &key);
```

Comment:
Make this a plain function instead of a method.
Then we can use `gt("key")` instead of `KiwixApp::get("key")`